src/utils/search/get-matches.js

```diff
@@ -15,7 +20,8 @@ export default function getMatches(
   const lines = text.split("\n");
   for (let i = 0; i < lines.length; i++) {
     let singleMatch;
-    while ((singleMatch = regexQuery.exec(lines[i])) !== null) {
+    const line = guardLineLength(lines[i]);
+    while ((singleMatch = regexQuery.exec(line)) !== null) {
```

Comment:
I understand what you mean. I am just afraid that it will feel broken and frustrating for the user. The best thing we could do is give back an error message saying "line too long". I will throw that together and lets see what happens.